src/views/CreateView.vue

```diff
@@ -343,6 +355,30 @@ export default {
         this.isFormSubmitted = true
       }
     }
+  },
+  async mounted() {
+    if (this.isSignedIn) {
+      const fetchExercises = async () => {
+        const response = await supabase.from('Exercises').select()
+        
+        if (response.error === null) {
+          const { data } = response
+
+          data.forEach((item) => {
+            const { img, muscles, name } = item.exercise
+            this.exercises.push({ img, muscles, name })
+          })
```

Comment:
Use map instead

Reply:
```js
data.map(({muscle, img, name}) => ({ muscle, img, name})

```

Reply:
This method returns an array, so you can assign it's result to exercises in the same line 😁

Reply:
I couldn't apply this the way you told me to because `muscle`, `img`, and `name` are properties of `data[x].exercise`, not of the data itself.

However, I changed it the way I could, please, let me know if this is ok. I won't merge this branch until I get the feedback from you regarding this matter.

Reply:
for one level nested prop I'd still consider using destruct.
```js
data.map(({ exercise: { muscle, img, name }}) => ({ muscle, img, name});
```